src/theme/config.ts

```diff
@@ -1,6 +1,10 @@
-import { ThemeConfig } from "@chakra-ui/react";
+import { ThemeConfig } from '@chakra-ui/react'
 
 export const config: ThemeConfig = {
   initialColorMode: 'light',
   useSystemColorMode: false
 }
+
+export const borderRadiusStandard: string = '6px'
+
+export const scrollbarRadiusStandard: string = '8px'
```

Comment:
Standardized Wooting styling applied

Reply:
It's kinda weird to do these as constants, there is a default theme configuration for a reason

Reply:
Theoretically you could add a new key to this: https://chakra-ui.com/docs/styled-system/theme#border-radius 
e.g. "default" and reference that instead of using a constant you have to import everywhere

Reply:
Tried simplifying: [3c1c493](https://github.com/WootingKb/wooting-macros/pull/194/commits/3c1c493f6325ac45eb159fbe7222f368a15ecf2e)